packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -1135,7 +1136,7 @@ export default class TradeStore extends BaseStore {
 
     get is_synthetics_trading_market_available() {
         return !!this.active_symbols?.find(
-            item => item.market === 'synthetic_index' && !isMarketClosed(this.active_symbols, item.symbol)
+            item => item.subgroup === 'synthetics' && !isMarketClosed(this.active_symbols, item.symbol)
```

Comment:
`subgroup` is not translated to other languages, correct? In case if user switch from EN to any lang

Reply:
@kate-deriv it's not translated on API side, it's always in ENG :) so it will work for all languages